### DISCUSSION TITLE
bat does not apply the theme.

### DISCUSSION THREAD

**Discussion by Manix80** [jarun/nnn#1734] [Q&A] [answered]
after countless tests i can't change the preview theme with preview-tui and bat.

there's nothing in the nnn folder (.config) except preview-tui. 

I'm relying on you to help me ...
config zsh :

[.zshrc](http://0x0.st/HVcP.txt)

.zprofile:
```
#!/usr/bin/env zsh

typeset -U path PATH
path=(~/.local/bin $path)
export PATH

# XDG
export XDG_CONFIG_HOME="${HOME}/.config"
export XDG_DATA_HOME="${HOME}/.local/share"
export XDG_CACHE_HOME="${HOME}/.cache"
export XDG_DOWNLOAD_DIR="${HOME}/Downloads"
#export XDG_DESKTOP_DIR="${HOME}/Downloads"

# DIVERS
export MOZ_ENABLE_WAYLAND=1
export GTK_THEME=Nordic
export EDITOR="/usr/bin/nvim"
export PAGER="/usr/bin/less"
export BROWSER="/usr/bin/firefox"
export GOPATH="${XDG_DATA_HOME}/go"
export RUSTUP_HOME="${XDG_DATA_HOME}/rustup"
export CARGO_HOME="${XDG_DATA_HOME}/cargo"
export NPM_CONFIG_USERCONFIG="${XDG_DATA_HOME}/npm/.npmrc"
export npm_config_prefix="${XDG_DATA_HOME}/npm"
export MANPAGER="sh -c 'col -bx | bat -l man -p'"
export MANROFFOPT="-c"
export LESSHISTFILE=-
export BAT_THEME="Dracula"
export ZSH_AUTOSUGGEST_HIGHLIGHT_STYLE='fg=239'
export DIFFPROG='nvim -d'
#export LESS='-~'
export NNN_PAGER="bat"
export NNN_OPTS='Hde'
export NNN_PLUG='p:preview-tui'
export NNN_ARCHIVE="\\.(7z|a|ace|alz|arc|arj|bz|bz2|cab|cpio|deb|gz|jar|lha|lz|lzh|lzma|lzo|rar|rpm|rz|t7z|tar|tbz|tbz2|tgz|tlz|txz|tZ|tzo|war|xpi|xz|Z|zip)$"
export NNN_TRASH=2
export NNN_FIFO='/tmp/nnn.fifo'
BLK="0B" CHR="0B" DIR="04" EXE="06" REG="00" HARDLINK="06" SYMLINK="06" MISSING="00" ORPHAN="09" FIFO="06" SOCK="0B" OTHER="06"
export NNN_FCOLORS="$BLK$CHR$DIR$EXE$REG$HARDLINK$SYMLINK$MISSING$ORPHAN$FIFO$SOCK$OTHER"

if [[ -z "$DISPLAY" ]] && [[ $(tty) = /dev/tty1 ]]; then
  exec Hyprland
fi
```

**Comment by Manix80** [comment]
problem solved by changing the line :
`"NNN_BATTHEME=${NNN_BATTHEME:-Nord}"`
in the preview-tui file ...
and `"NNN_BATSTYLE=${NNN_BATSTYLE:-plain}"`
if you don't want the numbers.
all I have to do now is try to close the preview when I open a file.
I'm going to sleep :)

**Comment by Manix80** [comment]
Hi
I'm coming back to you with another question, is it possible to modify the restorepreview shortcut?
I use the right arrow to open a file.
awesome nnn <3

**Comment by Manix80** [comment]
hey,
I added this line to nnn.h :
```
	/* Edit in EDITOR */
	{ 'e',            SEL_EDIT },
	{ KEY_RIGHT,      SEL_EDIT },
```
the 'e' key works with restorepreview but not the right arrow.
any ideas ?

**Reply by N-R-K** [reply]
I think that's the intended behavior. "open" key might open the file in some gui or whatever so `nnn` cannot tell if the preview plane should be closed or not in those cases.

This is why restore preview patch only works on "edit" key.

**Reply by Manix80** [reply]
ok thanks

**Comment by David-Else** [comment, marked as answer]
This works for me in the `.bashrc` file, no need to alter the plugin:

```bash
export NNN_BATTHEME="Visual Studio Dark+"
export NNN_BATSTYLE="plain"
```

**Reply by Manix80** [reply]
works too! thanks for the tips
